packages/thumbprint-react/components/BlockList/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styles from './index.module.scss';
+
+export function BlockListItemLink({ children, to, className }) {
```

Comment:
Instead of white-listing these three props, we could simply allow all props:

```jsx
export function BlockListItemLink({ className, ...rest }) {
    return (
        // eslint-disable-next-line jsx-a11y/anchor-has-content
        <a
            className={classNames({
                [styles.itemLink]: true,
                [className]: className !== undefined,
            })}
            {...rest}
        />
    );
}
```

The PropTypes can then become:

```js
BlockListItemLink.propTypes = {
    /**
     * URL pointing to the item link destination.
     */
    href: PropTypes.string.isRequired,
    /**
     * The content of the item link.
     */
    children: PropTypes.node.isRequired,
};
```

And examples would use `href` instead of `to`. I think this is a little better since this component is only concerned with the three lines of CSS that allow the developer to get a full clickable target.

Reply:
Fixed